tests/test_model.py

```diff
@@ -3463,6 +3471,86 @@ def test_split_model_part_expected_raise(self, create_default_2d_soil_material:
         with pytest.raises(ValueError, match=f"Model part type and new parameters type must match."):
             model.split_model_part("process_2d", "split_group", [1], create_default_2d_soil_material)
 
+    def test_finalise(self, create_default_2d_soil_material: SoilMaterial):
```

Comment:
can you rename it to something like test_finalise_json_output_raises

Reply:
Done!